### PR TITLE
Add fix menu

### PR DESCRIPTION
**Pull request by Nadolnij-Andrii**

### CODE REVIEW

File: frontend/ATS/src/app/users/components/registration-box/registration-box.component.ts [hunk 1]
```diff
@@ -49,4 +49,8 @@ export class RegistrationBoxComponent {
     ]),
   }, { validators: this.loginRegistCommonComponent.passwordsMatch });
 
+
+  onSubmit(){
+    console.log(this.registrationForm.controls['userFirstName'].valid);
```
Comment (Yaroslaww-1) [inline frontend/ATS/src/app/users/components/registration-box/registration-box.component.ts:54]: Do we need this ```console.log```?

File: frontend/ATS/src/app/users/user-routing.module.ts [hunk 2]
```diff
@@ -3,16 +3,18 @@ import { CommonModule } from '@angular/common';
...
 const routes: Routes = [
-  { path: 'home', component: HomeComponent },
   {
     path: 'login',
     pathMatch: 'full',
     component: LoginComponent,
   },
-  { path: 'registration', component: RegistrationComponent },
+  { 
+    path: 'registration',
```
Comment (Yaroslaww-1) [inline frontend/ATS/src/app/users/user-routing.module.ts:14]: Please use AppRoute enum here

File: frontend/ATS/src/app/users/components/registration-box/registration-box.component.ts (hunk 1 above)
Comment (Nadolnij-Andrii) [reply frontend/ATS/src/app/users/components/registration-box/registration-box.component.ts:54]: No. It was added for debugging.

**Comment by Nadolnij-Andrii** [comment]
New changes have been made in accordance with the comments.

**Review by anton-matsyshyn — APPROVED** [review]

**Pull request merged**